h2o-py/h2o/model/model_base.py

```diff
@@ -190,9 +190,9 @@ def predict_contributions(self, test_data, output_format="Original", top_n=None,
             contributions for 1-hot encoded features, specifying a Compact output format will produce a per-feature
             contribution. One of: ``"Original"``, ``"Compact"`` (default: ``"Original"``).
         :param top_n: Return only #top_n highest contributions + bias.
-        :param top_bottom_n: Return only #top_bottom_n lowest contributions + bias
-                             If top_n and top_bottom_n are defined together then return array of #top_n + #top_bottom_n + bias
-        :param abs_val: True to compare absolute values of contributions
+        :param bottom_n: Return only #bottom_n lowest contributions + bias
+                             If top_n and bottom_n are defined together then return array of #top_n + #bottom_n + bias
```

Comment:
many tests are using values like `top_n=-1` or `bottom_n=0`: could be document here the semantic of those particular values?
I think I understand `0`, although I don't see why it is used in tests: probably has the same semantic as `None`, in which case it's not necesseray to document that one, but what about `-1`?

Reply:
Good point! Thanks, I'll improve the documentation for Python and R